backend/src/main/java/datastorage/DataStore.java

```diff
@@ -59,4 +59,10 @@ public interface DataStore {
    * @param fileJson The file which is used to populate the database
    */
   public void parseFile(String fileJson);
+
```

Comment:
Sounds good. I was hoping to keep the firebase integration as simple as possible but this still seems doable. I'll probably be able to get started on firebase relatively soon.